Modelica/Resources/C-Sources/ModelicaStandardTablesUsertab.c

```diff
@@ -48,7 +48,7 @@
 #include "ModelicaUtilities.h"
 
 #if defined(DUMMY_FUNCTION_USERTAB)
-#if (defined(__clang__) || defined(__GNUC__)) && !(defined(__apple_build_version__) || defined(__MINGW32__) || defined(__MINGW64__) || defined(__CYGWIN__))
+#if (defined(__clang__) || defined(__GNUC__)) && !(defined(__apple_build_version__) || defined(__APPLE__) || defined(__MINGW32__) || defined(__MINGW64__) || defined(__CYGWIN__))
```

Comment:
This fix is independet of the CI enhancements and should go to a separate PR.

Reply:
I have moved it to #2950 . I expect this PR to be unstable until #2950 is merged.